types/culori/src/displayable.d.ts

```diff
@@ -1,3 +0,0 @@
-import { Color } from "./common";
-
-export default function displayable(color: Color | string): boolean;
```

Comment:
`displayable()` not deleted; just moved to `clamp.d.ts` because that’s also where it lives in the main package (figured it would make tracking type changes easier if they’re 1:1 with source files)